tasks.js

```diff
@@ -112,4 +112,38 @@ module.exports = [
       },
     }],
   },
+
+  // Symptomatic contact follow up
+  {
+    name: 'symptomatic_contact_follow_up',
+    icon: 'icon-healthcare',
+    title: 'task.symptomatic_contact_follow_up.title',
+    appliesTo: 'contacts',
+    appliesToType: undefined,
+    appliesIf: function (contact) {
+      this.mostRecentQuarantineFollowUp = Utils.getMostRecentReport(contact.reports, 'QUARANTINE_FOLLOW_UP');
+
+      return !!this.mostRecentQuarantineFollowUp &&
+        this.mostRecentQuarantineFollowUp.fields.symptoms_check === 'true';
+    },
+    resolvedIf: function (c, r, event) {
+      const startTime = Utils.addDate(event.dueDate(c, r), -event.start);
+      const endTime = Utils.addDate(event.dueDate(c, r), event.end + 1);
+
+      const reportsAfterQuarantineFollowUp = c.reports.filter(report => report.reported_date >= this.mostRecentQuarantineFollowUp.reported_date);
+      return Utils.isFormSubmittedInWindow(reportsAfterChaVerification, 'locator', startTime, endTime);
+    },
+    events: [{
+      start: 1,
+      end: 3,
+      dueDate: function() {
+        return Utils.addDate(new Date(this.mostRecentQuarantineFollowUp.reported_date), 1);
+      },
+    }],
+    actions: [{
+      type: 'report',
+      form: 'patient_screening',
```

Comment:
You should also include this form as part of this task. 
There should be only one question 

Have you contacted the symptomatic patient ?  ( yes/no )

Reply:
We already have a patient_screening form

Reply:
There are two things to consider : 
1. That form `patient_screening` is actually useless and don't answer any requirement for the moment. I keep it around just in case we can build on top of it for our symptoms screening use case that we should implement in 2 weeks. 
2. We do not intend to do a patient screening in this task ( we already know the patient is symptomatic ) this task is more to make sure the symptomatic patient have been contacted, please see question to ask in that form on my previous comment. You may also want to have a look at covid_rdt_followup form because it's kind of the same content we are looking to have

Reply:
Got it. Tracking here https://github.com/medic/config-covid/issues/52